src/dev/impl/DevToys/Views/Tools/EncodersDecoders/JwtDecoderEncoder/JwtDecoderControl.xaml

```diff
@@ -186,7 +198,45 @@
             CodeLanguage="json"
             IsReadOnly="True"
             AllowExpand="True"
-            ExpandedChanged="PayloadCodeEditor_ExpandedChanged"/>
+            ExpandedChanged="PayloadCodeEditor_ExpandedChanged">
+            <controls:CodeEditor.InfoContent>
+                <Border
+                    BorderThickness="1" 
+                    BorderBrush="{ThemeResource TextControlBorderBrush}" 
+                    CornerRadius="{ThemeResource ControlCornerRadius}"
+                    Background="{ThemeResource TextControlBackground}">
+                    <toolkit:DataGrid
```

Comment:
@btiteux unfortunately this won't be fun to do in Blazor 😅 Hopefully we can reuse something open source to help us, or come up with an easier to implement view.